modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -210,6 +212,38 @@
         y (h2x/->timestamp y)]
     (sql.qp/datetime-diff driver unit x y)))
 
+(def server-side-relative-datetime-units
+  "Units which, when used with :relative-datetime clause, cause computation server side. Those are
+   [[metabase.util.date-2/add-units]] coarser or equal to day. For more details see
+   [[server-side-relative-datetime-honeysql-form]]."
+  #{:day :week :month :quarter :year})
+
+(defn- use-server-side-relative-datetime?
+  [unit]
+  (contains? server-side-relative-datetime-units unit))
```

Comment:
This function doesn't seem particularly useful.

Reply:
I've decided to move definition of `server-side-relative-datetime-units` (from [your previous comment](https://github.com/metabase/metabase/pull/38604#discussion_r1484401169)) and its docstring into this function. This function is redefined in some tests to force the old behavior and compare the result with new one.